scss/components/_nav.scss

```diff
@@ -13,3 +13,29 @@
     }
   }
 }
+
+.navbar-brand {
+  color: $secondary;
+  &:before {
+    font-size: calc-em(35,20);
+    @include media-breakpoint-up(lg) {
+      font-size: calc-em(44,20);
+    }
+  }
+
+  &:hover {
+    color: $secondary;
+  }
+}
+
+.announcement-bar {
+  padding: calc-em(6) 0;
+  a {
+    font-size: 1.125em;
+  }
+}
+
+.cart-drawer {
```

Comment:
cart-drawer it's a separate compound and Lusia will develop it. Also I cannot see this code on some HTML file, where is it?
Please keep the order as we talked